### DISCUSSION TITLE
Is react-redux@8 the recommended upgrade?

### DISCUSSION THREAD

**Discussion by yoyo837** [reduxjs/react-redux#1936] [Q&A] [answered]
If we are still stuck on `react@16`, is `react-redux@8` the recommended upgrade?

**Comment by markerikson** [comment, marked as answer]
Yes, React-Redux v8 works with any version of React that has hooks (16.8+).  If you're on an earlier version of 16.x, you should be able to at least update to the last 16.x release (16.13, I think?) with no problems.

The main issue would then be TS types.  React-Redux v8 expects to work with `@types/react@18`, and other libraries may not be ready for that.  You also need to make sure there's only _one_ version of `@types/react` installed in a project, and `@types/react@18` made a change to `props.children` that you need to be ready to deal with.

**Reply by yoyo837** [reply]
Thanks for your reply.
